provider/workspace.go

```diff
@@ -98,6 +111,11 @@ func workspaceDataSource() *schema.Resource {
 				Computed:    true,
 				Description: "UUID of the workspace.",
 			},
+			"is_prebuild": {
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "Whether the workspace is a prebuild.",
```

Comment:
```suggestion
				Description: "Set to true if the workspace is a prebuild.",
```
Pet-peeve: Whether doesn't describe which is the expected value, even if it can be inferred from `is_`.